Scripts/cba.py

```diff
@@ -84,6 +84,15 @@ def read_scenario(path, time_period):
                 matrices[transport_class][mtx_type] = 0
             else:
                 matrices[transport_class][mtx_type] = numpy.array(files[mtx_type][ass_class])
+                if mtx_type == "cost":
+                    if transport_class == "transit_work":
+                        trips_per_month = numpy.full_like(matrices[transport_class][mtx_type], 60)
+                        # Surrounding area has a lower number of trips per month
+                        trips_per_month[901:, :] = 44
+                        trips_per_month = 0.5 * (trips_per_month+trips_per_month.T)
+                        matrices[transport_class][mtx_type] / trips_per_month
```

Comment:
I think the values need to be assigned to dictionary:
`matrices[transport_class][mtx_type] = matrices[transport_class][mtx_type] / trips_per_month`